redbot/core/core_commands.py

```diff
@@ -2077,6 +2078,31 @@ async def api(self, ctx: commands.Context, service: str, *, tokens: TokenConvert
         await ctx.bot.set_shared_api_tokens(service, **tokens)
         await ctx.send(_("`{service}` API tokens have been set.").format(service=service))
 
+    @api.command(name="list")
+    async def api_list(self, ctx: commands.Context):
+        """Show all external API services that have been set.
+
+        This command does not expose keys or tokens associated with services,
```

Comment:
Hmm, it would (IMO) be useful to also know the key names that were set for the API services (i.e. `twitch: client_id, client_secret`), but I'm not gonna request it here - it's up to you whether you want to include it.

Reply:
I will definitely look into that, but at a later date.

Reply:
Based on a conversation in Discord, this will be made as part of this PR due to needed changes in the API methods that are added by this PR.